configure.ac

```diff
@@ -734,6 +734,10 @@ if test x$TARGET_OS != xwindows; then
   AX_CHECK_COMPILE_FLAG([-fPIC],[PIC_FLAGS="-fPIC"])
 fi
 
+# All versions of gcc that we commonly use for building are subject to bug
+# https://gcc.gnu.org/bugzilla/show_bug.cgi?id=90348. To work around that, set
+# -fstack-reuse=none for all builds.
```

Comment:
Maybe mention that -fstack-reuse flag is gcc specific and won't affect other compilers like clang.